packages/directlinespeech/__tests__/utilities/fetchSpeechCredentialsWithCache.js

```diff
@@ -2,15 +2,28 @@ import fetch from 'node-fetch';
 
 const TOKEN_URL_TEMPLATE = 'https://{region}.api.cognitive.microsoft.com/sts/v1.0/issueToken';
 
-async function fromWaterBottle() {
+async function fromWaterBottle(enableInternalHttpSupport = false) {
   const res = await fetch('https://webchat-waterbottle.azurewebsites.net/token/speechservices');
 
   if (!res.ok) {
     throw new Error(`Failed to fetch Cognitive Services Speech Services credentials, server returned ${res.status}`);
   }
 
+  let directLineToken;
   const { region, token: authorizationToken } = await res.json();
 
+  if (enableInternalHttpSupport) {
+    const res2 = await fetch('https://webchat-waterbottle.azurewebsites.net/token/directline');
+
+    if (!res2.ok) {
+      throw new Error(`Failed to fetch Cognitive Services Direct Line credentials, server returned ${res2.status}`);
+    }
+
+    const { token: directLineToken } = await res2.json();
+
+    return { authorizationToken, region, directLineToken };
```

Comment:
Cleanliness nitpick: alphabetize the properties.

```suggestion
    return { authorizationToken, directLineToken, region };
```